src/foam/dao/CacheType.js

```diff
@@ -24,6 +24,10 @@ foam.ENUM({
     {
       name: 'TTL',
       documentation: "Using a TTL (Time to Live) cache"
+    },
```

Comment:
I think we should change the approach, since you might want both TTL and TTL_SELECT. Let's discuss.

Reply:
Updated according to our discussion - https://github.com/foam-framework/foam2/pull/3081/commits/b374a9001e495232aa53ba72fdd142c4d0eb9111.